README.md

```diff
@@ -1,5 +1,15 @@
 # Integreatly Operator
 
+---
+**NOTE**
```

Comment:
Could we also leave a similar note above the INSTALLATION_TYPE variable in the Makefile. I see that it's being left as managed (for now?). Just thinking that it should be clear that if you want to do anything in the makefile.

Reply:
sure